model/file_info.go

```diff
@@ -6,72 +6,169 @@ package model
 import (
 	"bytes"
 	"encoding/json"
+	"image"
 	"image/gif"
 	"io"
 	"mime"
 	"path/filepath"
+	"strings"
 )
 
 type FileInfo struct {
-	Filename        string `json:"filename"`
-	Size            int    `json:"size"`
+	Id              string `json:"id"`
+	UserId          string `json:"user_id"`
+	PostId          string `json:"post_id,omitempty"`
```

Comment:
Discussed this and decided to leave as is for now. It should be easy to migrate if we want to add enhanced files features.